pkgs/applications/networking/instant-messengers/discord/openasar.nix

```diff
@@ -1,16 +1,28 @@
-{ lib, stdenv, fetchFromGitHub, nodejs, bash, nodePackages }:
+{ lib, stdenv, fetchFromGitHub, nodejs, bash, nodePackages, unzip }:
 
 stdenv.mkDerivation rec {
-  version = "unstable-2022-06-10";
+  version = "unstable-2022-06-27";
   pname = "openasar";
 
   src = fetchFromGitHub {
     owner = "GooseMod";
     repo = "OpenAsar";
-    rev = "c6f2f5eb7827fea14cb4c54345af8ff6858c633a";
-    sha256 = "m6e/WKGgkR8vjKcHSNdWE25MmDQM1Z3kgB24OJgbw/w=";
+    rev = "6f7505fb91a07035d3661a3a7bf68b3018ddfd82";
+    sha256 = "2tb6OgYOnpryiyk7UH39sgzwtGJf9hNOpy74YqLI+Uk=";
   };
 
+  patchPhase = ''
+    runHook prePatch
+
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
Can't this break `prePatch` and `postPatch` calls :thinking: ?